connectors/aws-cert/src/test/java/com/vmware/connectors/aws/cert/AwsCertControllerTest.java

```diff
@@ -115,65 +121,45 @@ private ResultActions requestCards(String requestFile, String language) throws E
     // Request Cards
     /////////////////////////////
 
-    @Test
-    public void testRequestCardsSuccess() throws Exception {
+    @ParameterizedTest(name = "{index} ==> Language=''{0}''")
+    @DisplayName("Card request success cases")
+    @CsvSource({
+            "  , /awscert/responses/success/cards/card.json",
```

Comment:
Instead of ```  ```, you can use ```StringUtils.EMPTY``` here.

Reply:
Please remove it in all the places.

Reply:
Yes, this will be better for reading. Changed it all places.